stages/org.osbuild.containers.unit.create

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/python3
+import configparser
+import sys
+
+import osbuild.api
+
+
+def validate(filename, cfg):
+    # ensure the service name does not exceed maximum filename length
+    if len(filename) > 255:
+        raise ValueError(f"Error: the {filename} unit exceeds the maximum filename length.")
+
+    # Filename extension must match the config:
+    #   .service requires a Service section
+    #   .mount requires a Mount section
+    if filename.endswith(".container") and "Container" not in cfg:
+        raise ValueError(f"Error: {filename} unit requires Container section")
+    if filename.endswith(".volume") and "Volume" not in cfg:
+        raise ValueError(f"Error: {filename} unit requires Volume section")
+    if filename.endswith(".network") and "Network" not in cfg:
+        raise ValueError(f"Error: {filename} unit requires Network section")
+
+
+def main(tree, options):
+    filename = options["filename"]
+    cfg = options["config"]
+    validate(filename, cfg)
+
+    # We trick configparser into letting us write multiple instances of the same option by writing them as keys with no
+    # value, so we enable allow_no_value
+    config = configparser.ConfigParser(allow_no_value=True, interpolation=None)
```

Comment:
More of a note for myself but we could deduplicate some (probably most) of the code in this stage with the systemd unit so they only differ in schema.

That's for later though.